pkg/handlers/app.go

```diff
@@ -352,6 +354,12 @@ func mockReleaseToAppRelease(mockRelease integrationtypes.MockRelease) AppReleas
 	return appRelease
 }
 
+var testClientSet kubernetes.Interface
+
+func SetTestClientSet(clientset kubernetes.Interface) {
+	testClientSet = clientset
+}
+
```

Comment:
Thoughts on having this in the `k8sutil` package and the `if testClientset != nil` logic can be part of the `GetClientset()` function? That way it could be used without the need to if/else each time we want to potentially use the test client in a handler (or elsewhere).

Reply:
That is great suggestion Craig.  Really neat.

CC: @marccampbell

Reply:
My only concern is that we could need to return an `kubernetes.Interface` for GetClientSet() instead of the pointer type `*kubernetes.Clientset`

Reply:
Makes sense to me. I'm good with the PR as it is 👍